embeddedconfig/global.yaml

```diff
@@ -255,6 +255,18 @@ client:
         testurl: https://fronted-ping.dsa.akamai.getiantem.org/ping
         validator:
           rejectstatus: [403]
+        frontingsnis:
+          default:
+            usearbitrarysnis: false
+            arbitrarysnis:
+            - amazon.com
+            - mercadopago.com
+            - facebook.com
+            - instagram.com
+            - twitter.com
```

Comment:
Let's use the domains from [this list](https://github.com/getlantern/lantern-cloud/blob/175729685c665e2ae92e1c4e458d8fa2cc5d0dff/cmd/tlsmasq-tool/tmasqorigins/default.yaml) (I realize the ones you've got here now are just placeholders).